internal/testmodels/subschemas_switch.go

```diff
@@ -23,12 +23,17 @@ type BoolPayload struct {
 
 func (ex ExampleCase) Case() jsonschema.SchemaSwitch {
 	cases := make(map[string]interface{})
+	order := []string{}
 	cases["bool"] = BoolPayload{}
+	order = append(order, "bool")
 	cases["int"] = IntPayload{}
+	order = append(order, "int")
 	cases["string"] = StringPayload{}
+	order = append(order, "string")
 
 	return jsonschema.SchemaSwitch{
```

Comment:
This is a breaking change because users would have to supply this order and it wouldn't be entirely obvious why. If it's not supplied, this section of jsonschema isn't generated at all.

Why not generate this internally instead?

Reply:
Is there a case where the developer would want to actually control for order here? If so, we can make a case to expose it but then generate it internally if it's empty.

Reply:
I will make the changes, and write the tests for the same Update in PR.